src/Compilers/CSharp/Portable/Symbols/Metadata/PE/MetadataDecoder.cs

```diff
@@ -230,6 +230,11 @@ public static bool IsOrClosedOverATypeFromAssemblies(TypeSymbol symbol, Immutabl
 
                     do
                     {
+                        if (namedType.IsTupleType)
```

Comment:
The name of the method is odd. Although you haven't changed it consider renaming to ```ReferencesTypeFromAssemblies``` or something like that.